### ISSUE TITLE
Request confirmation before resetting quest selection configuration

### ISSUE THREAD

**Issue by waldyrious** [westnordost/StreetComplete#1095]
I just clicked "Reset" in the quest selection view, by mistake, and all the customizations I had done were instantly gone. Of course this is my own fault, but since this is a destructive action, it would be nice to get a confirmation dialog, to prevent such errors.

To make it extra nice, perhaps the app could also show a "toast" message confirming the deed, and offering a last chance to undo it (I believe toast messages can have an active button attached to them — at least I've seen that UI in another app)

**Comment by rugk** [comment]
With "reset" button you mean your phone's (hard- or soft-) back-button when you have a quest open?

**Comment by westnordost** [comment]
Yeah, these kind of "toasts" are called "snackbars"... Android naming 🙄 
To do this makes sense.

**Comment by waldyrious** [comment]
> With "reset" button you mean your phone's (hard- or soft-) back-button when you have a quest open?

Sorry I didn't reply earlier. I meant the "reset" link that shows up within the app in the top right of the "Quest selection" section of the settings.
